CI/bench/tabular/amlb_user_dir/benchmarks/small.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# Locally stored instead of fetched from openml to avoid server errors
+# Small datasets to test benchmarking - extracted from autogluon-bench/data/metadata/task_metadata.csv
+
+- name: arcene
+  openml_task_id: 190412
+
+- name: tecator
+  openml_task_id: 359934
+
+- name: boston
+  openml_task_id: 359950
```

Comment:
datasets should be changed to the following as they are the fastest to train on:

```
# binary
- name: Australian
  openml_task_id: 146818

# multiclass
- name: vehicle
  openml_task_id: 190146

# regression
- name: sensory
  openml_task_id: 359931
```